x/vm/types/scaling.go

```diff
@@ -79,27 +59,10 @@ func ConvertCoinsFrom18Decimals(coins sdk.Coins) sdk.Coins {
 	convertedCoins := make(sdk.Coins, len(coins))
 	for i, coin := range coins {
 		if coin.Denom == evmDenom {
-			evmCoinDecimals := GetEVMCoinDecimals()
-
-			newAmount := coin.Amount.Quo(evmCoinDecimals.ConversionFactor())
-
-			coin = sdk.Coin{Denom: coin.Denom, Amount: newAmount}
+			evmExtendedDenom := GetEVMCoinExtendedDenom()
+			coin = sdk.Coin{Denom: evmExtendedDenom, Amount: coin.Amount}
```

Comment:
The logic for actual amount handling has been moved to `preciseBank`, so `ConvertEvmCoinFrom18Decimals` and `ConvertCoinsFrom18Decimals` now only perform denom changes.
However, the function names still use `Convert`, and since the other above `Convert*` functions actually perform amount conversions, this breaks consistency and could lead to confusion or mistakes.

Reply:
I renamed that funcs to `ChangeEvmCoinDenomFrom18Decimals` and `ChangeCoinsDenomFrom18Decimals`.